pkg/controllers/hub/trafficmanagerprofile/controller_integration_test.go

```diff
@@ -185,6 +185,13 @@ var _ = Describe("Test TrafficManagerProfile Controller", func() {
 		It("Updating trafficManagerProfile spec to valid and validating trafficManagerProfile status", func() {
 			profile.Spec.MonitorConfig.IntervalInSeconds = ptr.To[int64](30)
 			profile.Spec.MonitorConfig.TimeoutInSeconds = ptr.To[int64](10)
+			// add custom headers
+			profile.Spec.MonitorConfig.CustomHeaders = []fleetnetv1beta1.MonitorConfigCustomHeader{
```

Comment:
why I don't seem to see this value being verified on the ATM resource?

Reply:
in the integration test, we're using a fake atm endpoint. It will be validated in the e2e tests.